platinum-bluetooth-characteristic.html

```diff
@@ -358,6 +390,42 @@
       },
 
       /**
+        * Save value when characteristic value changed in a notification session.
+        **/
+      _handleValueChanged: function(event) {
+        // In Chrome 50+, a DataView is returned instead of an ArrayBuffer.
+        this.value = event.target.value.buffer ? event.target.value : new DataView(event.target.value);
+      },
+
+      /**
+       * Returns a promise that will resolve when Bluetooth GATT Characteristic
+       * notification session starts.
+       *
+       * @return {Promise<void>}
+       */
+      _startNotifications: function() {
+        if (!this._characteristic) {
+          return Promise.reject(new Error('Bluetooth GATT Characteristic is not connected.'));
+        }
+        this._characteristic.addEventListener('characteristicvaluechanged', this._handleValueChanged.bind(this));
+        return this._characteristic.startNotifications();
+      },
+
+      /**
+       * Returns a promise that will resolve when Bluetooth GATT Characteristic
+       * notification session stops.
+       *
+       * @return {Promise<void>}
+       */
+      _stopNotifications: function() {
+        if (!this._characteristic) {
+          return Promise.reject(new Error('Bluetooth GATT Characteristic is not connected.'));
+        }
+        this._characteristic.removeEventListener('characteristicvaluechanged', this._handleValueChanged.bind(this));
```

Comment:
This won't remove the listener, since `this._handleValueChanged.bind(this)` returns a different value the second time it runs. I think I would just bind that listener once when this._characteristic is set, and leave it set.